CHANGELOG.md

```diff
@@ -26,6 +26,18 @@ CHANGELOG
 Swift Next
 ----------
 
+* [SE-266](https://github.com/apple/swift-evolution/blob/master/proposals/0266-synthesized-comparable-for-enumerations.md):
```

Comment:
The URL should be put at the bottom I think...

Reply:
fixed